src/HonzaBotner.Discord.Services/Commands/MessageCommands.cs

```diff
@@ -164,10 +165,30 @@ public RoleBindingsCommands(IRoleBindingsService roleBindingsService, ILogger<Ro
                     }
                     catch (Exception e)
                     {
-                        _logger.LogWarning(e, "Couldn't delete reaction for emoji: {0}", emoji.Name);
+                        _logger.LogError(e, "Couldn't add reaction for emoji: {0} on {1}",
+                            emoji.Name, url);
                     }
                 }
             }
+
+            [Command("dump")]
+            public async Task Dump(CommandContext ctx, [Description("URL of the message.")] string url)
```

Comment:
Can we make `url` optional so, without it, it will output all configurations?

Reply:
![image](https://user-images.githubusercontent.com/5287596/108569005-70d87300-730b-11eb-8752-fab54c42396a.png)

Reply:
We could and we should. Yes. Will do it.

Reply:
I will leave the list subcommand as not implemented and we will come back to this later.